libursa/src/cl/verifier.rs

```diff
@@ -204,7 +215,7 @@ impl ProofVerifier {
     ///                                      None).unwrap();
     /// assert!(proof_verifier.verify(&proof, &proof_request_nonce).unwrap());
     /// ```
-    pub fn verify(&self, proof: &Proof, nonce: &Nonce) -> UrsaCryptoResult<bool> {
+    pub fn verify(&mut self, proof: &Proof, nonce: &Nonce) -> UrsaCryptoResult<bool> {
```

Comment:
Why is self mutable during verify?

Reply:
Because during verify, `common_attributes` are update from `m_hat`s from each subproof.